### PR TITLE
Fix issues how to generate encrypted passwords

### LINKED ISSUE #451: Documentation: multiple issues in code example for generating encrypted passwords
I found multiple issues using `  ``python -c 'import crypt; print(crypt.crypt("My Password", "$6$My Sault"))'``  ` from [documentation](https://github.com/pykickstart/pykickstart/blob/master/pykickstart/commands/user.py#L179) to create encrypted passwords on Fedora 37 
* First of all: it's not working, due to a space in "My Sault"
* "Sault" is a typo
* Salt is not randomised; people might reuse it due to blindly copy and pasting it
* Password could be readable in shell history
* crypt() is deprecated and slated for removal in Python 3.13
* SHA-512 is used while yescrypt is the [default since Fedora 35](https://fedoraproject.org/wiki/Changes/yescrypt_as_default_hashing_method_for_shadow) and supported [from Fedora 29 and RHEL 9 upwards](https://www.openwall.com/yescrypt/)
* Confusing and unnecessary quotes: \`` at the beginning and at the end.

### PR DESCRIPTION
**Pull request by Woi**
Introduce a better option but keep a fixed version of the old code example for older distributions Fixes #451

### CODE REVIEW

File: pykickstart/commands/user.py [hunk 1]
```diff
@@ -174,11 +174,17 @@ def _getParser(self):
                         password argument is assumed to already be encrypted.
                         ``--plaintext`` has the opposite effect - the password
                         argument is assumed to not be encrypted. To create an
-                        encrypted password you can use python::
+                        encrypted password you can use::
 
-                        ``python -c 'import crypt; print(crypt.crypt("My Password", "$6$My Sault"))'``
+                        mkpasswd -m yescrypt
```
Comment (bcl) [inline pykickstart/commands/user.py:179]: This should be `mkpasswd -m sha512crypt` for consistency.

**Review by bcl — CHANGES_REQUESTED** [review]
Thanks, just change yescrypt to sha512crypt and it's good.

File: pykickstart/commands/user.py (hunk 1 above)
Comment (bcl) [reply pykickstart/commands/user.py:179]: Somehow I missed we'd changed to yescrypt. Nevermind :)

**Review by bcl — APPROVED** [review]
Looks good, thanks!

**Comment by Woi** [comment]
Nice, thanks :)

**Pull request merged**